mdal/mdal_utils.cpp

```diff
@@ -505,6 +506,12 @@ void MDAL::addBedElevationDatasetGroup( MDAL::Mesh *mesh, const Vertices &vertic
   mesh->datasetGroups.push_back( group );
 }
 
+void MDAL::addBedElevationDatasetGroup( MDAL::Mesh *mesh, const Vertices &vertices )
```

Comment:
I would just use Bed elevation, not Terrain elevation . It is standard

Reply:
For me "Bed Elevation" is specific for river bed elevation. Terrain Elevation could be more generic. As TIN are note only for river, that's why I purpose to keep Bed Elevation for format specific to hydraulic modeling, and maybe we can purpose another name for other format not specific to river.

Reply:
whatabout keeping the Bed elevation and just change the name in the TIN driver...?